DigitalOcean.API/Models/Requests/UpdateDomainRecord.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Newtonsoft.Json;
+
+namespace DigitalOcean.API.Models.Requests {
+    public class UpdateDomainRecord {
+        /// <summary>
+        /// The record type (A, MX, CNAME, etc).
+        /// </summary>
+        [JsonProperty("type")]
+        public string Type { get; set; }
+
+        /// <summary>
+        /// The host name, alias, or service being defined by the record.
+        /// </summary>
+        [JsonProperty("name")]
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Variable data depending on record type.
+        /// </summary>
+        [JsonProperty("data")]
+        public string Data { get; set; }
+
+        /// <summary>
+        /// The priority of the host (for SRV and MX records. null otherwise).
+        /// </summary>
+        [JsonProperty("priority")]
+        public int? Priority { get; set; }
+
+        /// <summary>
+        /// The port that the service is accessible on (for SRV records only. null otherwise).
+        /// </summary>
+        [JsonProperty("port")]
+        public int? Port { get; set; }
+
+        /// <summary>
+        /// Time to live for the record, in seconds. If not set, the default value is 1800.
```

Comment:
Doc Update.

```
This value is the time to live for the record, in seconds. This defines the time frame that clients can cache queried information before a refresh should be requested. There is a minimum ttl value of 30, unless it is not set. If not set, the default value is the value of the SOA record. For SOA records, defines the time to live for purposes of negative caching.
```

Reply:
Updated docs for both `DomainRecord` and `UpdateDomainRecord`. The Data field also had some new notes in docs as well.